### ISSUE TITLE
utils.federated_avg() gives ar error with resnet models

### ISSUE THREAD

**Issue by Hjeljeli** [OpenMined/PySyft#4481]
## Description
When I call **utils.federated_avg()** on **resnet** models, I have the following error: __init__() missing 2 required positional arguments: 'block' and 'layers'. Any idea how to fix this?

## System Information
 - Pysyft Version: 0.2.9

## Code snippet
```
import torch
from torchvision import datasets, transforms, models

import syft as sy
from syft.workers.virtual import VirtualWorker
from syft.frameworks.torch.fl import utils
hook = sy.TorchHook(torch)

alice = VirtualWorker(id="alice", hook=hook)
bob = VirtualWorker(id="bob", hook=hook)
carol = VirtualWorker(id="carol", hook=hook)
workers = [alice, bob, carol]

model_in = models.resnet50(pretrained=True)

local_models = {}
for worker in workers:
    local_models[worker]= model_in.copy()
    
# Average the models
model_out = utils.federated_avg(local_models)
```

**Comment by steph-en-m** [comment]
@Hjeljeli  Could you also provide a code snippet to reproduce the error. Thanks

**Comment by Hjeljeli** [comment]
Sure. I added a code snippet to the description.

**Comment by Hjeljeli** [comment]
> This issue has been marked stale because it has been open 30 days with no activity. Leave a comment or remove the `stale` label to unmark it. Otherwise, this will be closed in 7 days.

Updating so this issue is active.

**Comment by cereallarceny** [comment]
Hello! Just letting you know that we are no longer planning on supporting anything on the 0.2.x product line and that all work should be ported over to 0.3.x, which is considered a complete rebuild of PySyft. Because of that, I'll be closing this issue. If you feel this is a mistake, or if the issue actually applies to 0.3.x as well, please feel free to ping me on Slack and I'll reopen the issue.
